nms/app/packages/magmalte/app/state/lte/SubscriberState.js

```diff
@@ -212,6 +213,18 @@ export async function setSubscriberState(props: SubscriberStateProps) {
     setSessionState(newSessionState.sessionState);
     return;
   }
+  if (Array.isArray(value)) {
```

Comment:
Based on above comment, if value is always an array we can eliminate this check and unify the logic. If a user passes in empty array, then we can consider that as a delete option. We might also in implement bulk delete in orc8r.

Reply:
Resolved